cypress_test/integration_tests/desktop/impress/top_toolbar_spec.js

```diff
@@ -63,6 +63,19 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'text-decoration', 'line-through');
 	});
 
+	it('Apply font color on text shape.', function() {
+		cy.get('#tb_editbar_item_fontcolor')
+			.click();
+
+		impressHelper.triggerNewSVGForShapeInTheCenter();
+
+		cy.get('.leaflet-pane.leaflet-overlay-pane g.Page .TextPosition tspan')
+			.should('have.attr', 'fill', 'rgb(0,0,0)');
+
+		cy.get('.leaflet-pane.leaflet-overlay-pane g.Page .TextParagraph')
+			.should('not.have.attr', 'font-color');
```

Comment:
I guess this test is not finished yet, right? The title says this test is about applying font color, but I can't see any assertion about some color was applied. So let's mark this change as something that still needs some changes.
One tip: When you reuse an existing test, you should be aware that we all humans, making mistakes, so when you copy some code from other places, you should check whether this code is correct or not.

Reply:
@tzolnai Is there an example or documentation that I can analyze to understand what kind of test I should implement for each test I'm doing?

I tested it with different color, checked it on the inspector on the way you guided me to do so. So, I could see the color was showing up, as you can see on the print below. Then I replace the color I used with the black color as I saw on the (apply_font_shape_spec.js) you pointed me to look.


![Screenshot from 2020-11-19 07-22-48](https://user-images.githubusercontent.com/69675010/99663466-6e812700-2a45-11eb-9c64-cff31512a602.png)




I added the 'not.have.attr' because I saw it on the file (apply_font_shape_spec.js) you pointed me to look. I wasn't aware of its need but I assumed I should add it because it was on the tests for mobile.

Reply:
When I try to add the classes according to my understanding of how it is supposed to be, the tests failed as the print below:

![Screenshot from 2020-11-19 09-15-59](https://user-images.githubusercontent.com/69675010/99665217-1697ef80-2a48-11eb-8a0a-1ff149c0a56a.png)


or, when I include the color code with a non-black color, the tests also failed:



![Screenshot from 2020-11-19 09-14-30](https://user-images.githubusercontent.com/69675010/99665226-18fa4980-2a48-11eb-8770-7d35f15a4f2f.png)

Reply:
![Screenshot from 2020-11-19 09-39-24](https://user-images.githubusercontent.com/69675010/99667481-409ee100-2a4b-11eb-911c-1f5feab7bc47.png)